refbox/src/tournament_manager/mod.rs

```diff
@@ -1020,9 +1030,13 @@ impl TournamentManager {
         );
     }
 
-    pub fn would_end_game(&self, now: Instant) -> Result<bool> {
-        if self.current_period == GamePeriod::SuddenDeath {
-            Ok(self.get_scores().are_not_equal())
+    pub fn could_end_game(&self, now: Instant) -> Result<bool> {
+        info!("could_end_game was called");
+        info!("Clock State is: {:?}", self.clock_state);
+        info!("Period: {:?}", self.current_period);
+        info!("Paused: {:?}", self.time_pause_confirmation);
```

Comment:
Frequently repeated log messages like this should be debug or trace, or they can be removed